pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -175,6 +177,12 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			}
 		}
 
+		secret := &corev1.Secret{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: crashCollectorKeyName, Namespace: namespace}, secret)
+		if err != nil {
```

Comment:
debug log sounds good...not sure of how to use IsNotFound here as most of the time(until mons are in quorum) it will through error when secret is not found.